INFO:bamboo.workflow:Worker processing sample MuonF with module /afs/cern.ch/work/w/wajid/METStudies/JMEFrameWrk/jme-validation/modules/DYModule.py. Inputs: ['root://cms-xrd-global.cern.ch//store/data/Run2022F/Muon/NANOAOD/16Dec2023-v1/2550000/cb8638ec-a51b-494e-bdc6-254666b48971.root', 'root://cms-xrd-global.cern.ch//store/data/Run2022F/Muon/NANOAOD/16Dec2023-v1/2550000/90138c3b-43fd-4311-9c3c-b1dbfd31b55f.root', 'root://cms-xrd-global.cern.ch//store/data/Run2022F/Muon/NANOAOD/16Dec2023-v1/2550000/c240846c-7f5e-49a8-a680-e039144276c4.root'], tree: Events, output: MuonF.root, certifiedLumiFile=None, runRange=None
TClass::Init:0: RuntimeWarning: no dictionary for class edm::Hash<1> is available
TClass::Init:0: RuntimeWarning: no dictionary for class edm::ParameterSetBlob is available
TClass::Init:0: RuntimeWarning: no dictionary for class edm::ProcessHistory is available
TClass::Init:0: RuntimeWarning: no dictionary for class edm::ProcessConfiguration is available
TClass::Init:0: RuntimeWarning: no dictionary for class pair<edm::Hash<1>,edm::ParameterSetBlob> is available
WARNING:bamboo.treedecorators:No branch name starting with Pileup_ in the tree - skipping group
WARNING:bamboo.treedecorators:nGenJet is not a branch in the tree - skipping collection
WARNING:bamboo.treedecorators:nGenVisTau is not a branch in the tree - skipping collection
WARNING:bamboo.treedecorators:nJetCHS is not a branch in the tree - skipping collection
WARNING:bamboo.treedecorators:nGenJetAK8 is not a branch in the tree - skipping collection
WARNING:bamboo.treedecorators:nSubGenJetAK8 is not a branch in the tree - skipping collection
WARNING:bamboo.treedecorators:GenJet is not a branch in the tree - skipping collection
INFO:bamboo.analysismodules:798 plots defined for MuonF in 1.66s, max RSS: 670.41MB
INFO:bamboo.workflow:Start backend graph construction
INFO:bamboo.dataframebackend:Number of uses per node type: 763 Histo1D, 3 Filter, 13 gInterpreter_Declare, 498 Define, 40 Histo2D
INFO:bamboo.workflow:Backend graph construction done in 25.13s, max RSS: 960.55MB
INFO:bamboo.workflow:Starting to fill plots (and skims)
Error in <TNetXNGFile::Open>: [ERROR] Invalid redirect URL
TTreeReader::SetEntryBase():0: RuntimeWarning: There was an issue opening the last file associated to the TChain being processed.
Traceback (most recent call last):
  File "/afs/cern.ch/work/w/wajid/METStudies/JMEFrameWrk/jme-validation/bamboovenv/bin/bambooRun", line 8, in <module>
    sys.exit(main())
  File "/afs/cern.ch/work/w/wajid/METStudies/JMEFrameWrk/jme-validation/bamboovenv/lib/python3.9/site-packages/bamboo/scripts/bambooRun.py", line 75, in main
    modInst.run()
  File "/afs/cern.ch/work/w/wajid/METStudies/JMEFrameWrk/jme-validation/bamboovenv/lib/python3.9/site-packages/bamboo/analysismodules.py", line 311, in run
    run_worker(self)
  File "/afs/cern.ch/work/w/wajid/METStudies/JMEFrameWrk/jme-validation/bamboovenv/lib/python3.9/site-packages/bamboo/workflow.py", line 523, in run_worker
    processOne(
  File "/afs/cern.ch/work/w/wajid/METStudies/JMEFrameWrk/jme-validation/bamboovenv/lib/python3.9/site-packages/bamboo/workflow.py", line 480, in processOne
    backend.runGraph()
  File "/afs/cern.ch/work/w/wajid/METStudies/JMEFrameWrk/jme-validation/bamboovenv/lib/python3.9/site-packages/bamboo/dataframebackend.py", line 923, in runGraph
    h.GetEntries()  # trigger event loop
cppyy.gbl.std.runtime_error: TH1D& ROOT::RDF::RResultPtr<TH1D>::operator*() =>
    runtime_error: An error was encountered while processing the data. TTreeReader status code is: 5
